ledger-tool/tests/basic.rs

```diff
@@ -37,66 +31,49 @@ fn bad_arguments() {
         .success());
 }
 
-fn nominal_test_helper(ledger_path: &str, ticks: usize) {
-    let meta_lines = 2;
-    let summary_lines = 1;
-
+fn nominal_test_helper(ledger_path: &str) {
     let output = run_ledger_tool(&["-l", ledger_path, "verify"]);
     assert!(output.status.success());
 
-    // Print everything
-    let output = run_ledger_tool(&["-l", ledger_path, "print", "-vvv"]);
+    let output = run_ledger_tool(&["-l", ledger_path, "print", "-vv"]);
     assert!(output.status.success());
-    assert_eq!(
-        count_newlines(&output.stdout)
-            .saturating_sub(meta_lines)
-            .saturating_sub(summary_lines),
-        ticks
-    );
```

Comment:
I don't think we need to be counting the number of lines of output from the command; we don't claim stability for this kind of output so I think confirming success is sufficient